doc/sphinx/user/install/local-installation/obtaining.md

```diff
@@ -22,7 +22,12 @@ supply cmake with the location:
 
 This will create an "out-of-source" build, where the build
 directory is different from the source directory. While in-source builds
-(where you run `cmake .` in your source directory), are supported, we strongly
+(where you run `cmake .` in your source directory) are supported, we strongly
 recommend an out-of-source build as described above. Specifically, running the
-whole test suite (see {ref}`sec:running_tests`46]) is only
+whole test suite (see {ref}`sec:running_tests`) is only
```

Comment:
The page is called `running-tests`. Can you try changing `{ref}`sec:running_tests` into `{ref}`sec:running-tests`?

Reply:
This needs to be fixed when the correct links are established.

Reply:
oke, I am fine with this being merged as is, since it is an improvement. The references can then be fixed in a follow up pull request. 

/rebuild